src/native/corehost/test/mockhostfxr/mockhostfxr.cpp

```diff
@@ -27,3 +27,27 @@ SHARED_API int HOSTFXR_CALLTYPE hostfxr_main_startupinfo(const int argc, const p
 
     return StatusCode::Success;
 }
+
+#ifdef MOCKHOSTFXR_5_0
+SHARED_API hostfxr_error_writer_fn HOSTFXR_CALLTYPE hostfxr_set_error_writer(hostfxr_error_writer_fn error_writer)
+{
+    return trace::set_error_writer(error_writer);
+}
+
+SHARED_API int HOSTFXR_CALLTYPE hostfxr_main_bundle_startupinfo(const int argc, const pal::char_t* argv[], const pal::char_t* host_path, const pal::char_t* dotnet_root, const pal::char_t* app_path, int64_t bundle_header_offset)
+{
+    trace_hostfxr_entry_point(_X("hostfxr_main_bundle_startupinfo"));
+
+    const pal::string_t dotnet_folder = get_directory(dotnet_root);
+
+    if (pal::strcmp(dotnet_folder.c_str(), _X("mockhostfxrBundleVersionFailure")))
```

Comment:
nit: this one uses `mockhostfxr` as the prefix for the known folder name and the above uses `hostfxr` - can we stick with one or the other?

Reply:
It should be mockhostfxr ideally